test/kaffe/group_member/worker/worker_test.exs

```diff
@@ -0,0 +1,44 @@
+defmodule Kaffe.WorkerTest do
+
+  use ExUnit.Case
+
+  alias Kaffe.Worker
+  
+  defmodule TestListHandler do
+    def handle_messages(messages) do
+      Enum.each messages, fn (m) ->
+        send :test_case, {:handle_messages, m.message}
+        :ok
+      end
+    end
+  end
+
+  defmodule TestSingleHandler do
+    def handle_message(message) do
+      send :test_case, {:handle_message, message.message}
+      :ok
+    end
+  end
+
+  test "use list handler" do
+    Process.register(self(), :test_case)
+    {:ok, worker_pid} = Worker.start_link(TestListHandler, 0)
+    Worker.process_messages(worker_pid, [%{message: :one}, %{message: :two}])
+
+    :timer.sleep 100
+
+    assert_received {:handle_messages, :one}
```

Comment:
I was thinking this test would look a little bit different from the "use single handler test". I wonder if there'd be value in having the `TestListHandler` send the full list of `messages` in one call rather than [enumerating over the list](https://github.com/spreedly/kaffe/pull/21/files#diff-2f4219351edaffa278a3e46fcfd65ea8R9).